packages/features/ee/workflows/components/AddActionDialog.tsx

```diff
@@ -202,8 +203,7 @@ export const AddActionDialog = (props: IAddActionDialog) => {
                 </>
               ) : (
                 <div className="mt-5">
-                  <Label>{t("sender_name")}</Label>
-                  <Input type="text" placeholder={SENDER_NAME} {...form.register(`senderName`)} />
+                  <TextField label={t("sender_id")} placeholder={SENDER_ID} {...form.register("senderId")} />
```

Comment:
This is not correct.

Reply:
why?, can u describe it more?

Reply:
```suggestion
                  <TextField label={t("sender_id")} placeholder={SENDER_NAME} {...form.register("senderName")} />
```
@emrysal is this what you were referring too?

Reply:
Cool, will make the changes